meta/SaiInterface.cpp

```diff
@@ -197,3 +197,29 @@ sai_status_t SaiInterface::get(
             return SAI_STATUS_FAILURE;
     }
 }
+
+sai_status_t SaiInterface::mdioRegRead(
+        _In_ sai_object_id_t switch_id,
+        _In_ uint32_t device_addr,
+        _In_ uint32_t start_reg_addr,
+        _In_ uint32_t number_of_registers,
+        _In_ bool clause_22,
+        _Out_ uint32_t *reg_val)
+{
+    SWSS_LOG_ENTER();
+
+    return SAI_STATUS_FAILURE;
+}
+
+sai_status_t SaiInterface::mdioRegWrite(
+        _In_ sai_object_id_t switch_id,
```

Comment:
if thoes apis are sai_switch_mdio_read_fn and sai_switch_mdio_write_fn, please use correspinding names switchMdioRead and switchMdioWrite for naming, also pleae remove parameter clause_22, sinice originally those apis in saiswitch.h dont have this parameter, and SaiInterface should mimic SAI apis interface

Reply:
there should be 2 sets of api

Reply:
Changed according to your comment.